src/promptflow-devkit/promptflow/_sdk/_utils.py

```diff
@@ -1059,6 +1059,7 @@ def entry_string_to_callable(entry_file, entry) -> Callable:
         try:
             module_name, func_name = entry.split(":")
             module = importlib.import_module(module_name)
+            importlib.reload(module)
```

Comment:
why alwys reload?
add comment why doing this

Reply:
Add comment.
It's used to avoid changes in the loaded module code, reload here to get the latest module.

Reply:
this function is no longer used now O O  @lalala123123  we now go through PythonInspector and gen meta in subprocess

Reply:
Not sure if always reload will trigger some issue. @elliotzh  I think the reload should happen in below scenarios:
- when debug server reload
- notebook kernel restart

Reply:
we don't need reload now.

if entry is a callable, it's already loaded in current process and we should respect that
if entry is a string, we not will load it in a subprocess so no need to reload anything